mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/utils/TableEntityTypeMappingInfo.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.mongodb.utils;
+
+import java.util.Map;
+
+/**
+ * @author Sergey Chernolyas &amp;sergey_chernolyas@gmail.com&amp;
+ */
+public class TableEntityTypeMappingInfo {
```

Comment:
Hi @DavideD !
The class contains mapping info (table <-> entity class)  because I need entity class for find OptionsContext.
```java
private static OptionsContext getPropertyOptions( OptionsService optionService, Class entityClass, String fieldName) {
		return optionService.context().getPropertyOptions( entityClass, fieldName );
	}
```
if it wrong ... How I can find the mapping?